build.cake

```diff
@@ -554,12 +554,24 @@ Task("SetPackageVersions")
 Task("SRGen")
 	.Does(() =>
 {
+    // save current working directory to restore at end of task
+    var taskStartedWorkingDirectory = System.IO.Directory.GetCurrentDirectory();
+    var relativePathPrefix = "." + System.IO.Path.DirectorySeparatorChar.ToString();
+
     var projects = System.IO.Directory.GetFiles(sourceFolder, "*.csproj", SearchOption.AllDirectories).ToList();
     var locTemplateDir = System.IO.Path.Combine(sourceFolder, "../localization"); 
```

Comment:
Hm, did you verify this works on macs? Anyways might be good to fix it up and just make it

```
Path.Combine(sourceFolder, "..", "localization");
```

Reply:
It's a mac-related fix 😄 so definitely tested there!